thoth/metrics_exporter/jobs/base.py

```diff
@@ -38,11 +38,7 @@
 @decorator
 def register_metric_job(method: Callable, *args, **kwargs) -> None:
     """A decorator for adding a metric job."""
-    start_time = time.monotonic()
-    try:
-        method(*args, **kwargs)
-    finally:
-        _LOGGER.info("Gathering metrics by job %r took %g seconds", method.__name__, time.monotonic() - start_time)
+    method(*args, **kwargs)
```

Comment:
you effectively removed the function of the decorator?!

Reply:
I removed try-except block. The measurement is done at https://github.com/thoth-station/metrics-exporter/blob/e822c98a89c97ba4f13bae151665481c4eddec92/wsgi.py#L85